### PR TITLE
SITES-521 whitelist email domains to *.gov.au

### PR DESCRIPTION
**Pull request by freshtonic**

### CODE REVIEW

**Comment by maxious** [comment]
LGTM
(the only exception I know of is csiro.au but later problem)

**Comment by ltankey** [comment]
Will this fail validation on admin?

**Comment by freshtonic** [comment]
@ltankey yes - and that means we'll need to change the admin login.

File: app/validators/email_domain_whitelist_validator.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class EmailDomainWhitelistValidator < ActiveModel::EachValidator
+
+  def validate_each(record, attribute, value)
+    record.errors.add(attribute, 'only *.gov.au email addresses permitted') if value !~ /\.gov\.au$/
```
Comment (tessereth) [inline app/validators/email_domain_whitelist_validator.rb]: Should it be `\z` instead of `$`?

Comment (freshtonic) [inline app/validators/email_domain_whitelist_validator.rb]: Yes it should.

Comment (freshtonic) [inline app/validators/email_domain_whitelist_validator.rb]: Fixed.

**Comment by ltankey** [comment]
LGTM

File: app/validators/email_domain_whitelist_validator.rb (hunk 1 above)
Comment (micapam) [inline app/validators/email_domain_whitelist_validator.rb]: Out of interest - what's the difference? I always use `$`.

**Pull request merged**